### PR TITLE
Task/tap 2520 add events documentation signal r

### PR DESCRIPTION
**Pull request by PatrickMira**

### CODE REVIEW

File: SignalR/2020.1/docs/API.md [hunk 1]
```diff
@@ -0,0 +1,192 @@
...
+  - [Subscribe to live sessions](#subscribe-to-live-sessions)
+  - [Unsubscribe from live sessions](#unsubscribe-from-live-sessions)
+
+The service is built using SignalR, therefore we suggest the use of the an [official Microsoft library](https://docs.microsoft.com/en-us/aspnet/core/signalr/javascript-client) for SignalR. The rest of the documentation will assume you're using one of these libraries.
+
+# Authentication
+The service is authenticating against the OAuth provider defined in [service config](ServiceConfig.md). Examples of how to pass token to SignalR service can be found [here](https://docs.microsoft.com/en-us/aspnet/core/signalr/authn-and-authz).
+
+# Telemetry Data Hub
+The telemetry data bub is the only hub available on the API. You can subscribe to various endpoints to be notified about new or closing streams. Also, you can subscribe to new parameter data notifications.
```
Comment (michaelgriffiths-mclaren) [inline SignalR/2020.1/docs/API.md]: Typo: 'bub' 😆

File: SignalR/2020.1/docs/API.md [hunk 2]
```diff
@@ -0,0 +1,192 @@
...
+## Subscribe to session update
+Notifies about session metadata or state change.
+
+#### Session Model
+| Property | Description |
+|-|-|
+| topicName | Name of the topic the session is streaming on |
+| streamId | The unique id of this stream |
+| sessionName | The name of the session, also known as Identifier in TAP |
+sessionDetails | Detail name (string) to detail value (string) dictionary |
```
Comment (michaelgriffiths-mclaren) [inline SignalR/2020.1/docs/API.md]: Missing start of column `|`

File: SignalR/2020.1/docs/API.md [hunk 3]
```diff
@@ -0,0 +1,192 @@
...
+| state | State of the session as string. Possible values are `Unknown`, `Waiting`, `Open`, `Closed`, `Truncated`, `Failed`, `Abandoned` |
+| sessionStart | DateTime using ISO 8601 |
+| dataState | Parameter data state as a string. Possible values:<br>`NoData`: Session is live but no data yet <br>`Interrupted`: Session is live and had data, but not in last 10 seconds<br>`Running`: Session is live and is streaming data <br> `Closed`: Session is closed
```
Comment (michaelgriffiths-mclaren) [inline SignalR/2020.1/docs/API.md]: Missing end of column `|`

File: SignalR/2020.1/docs/ServiceConfig.md [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+}
+```
+
+</details>
+
+#### Properties
+
+| Property  | Description | Example |  
+|-|-|-|
+| Server | The API server configuration. [See below](#server-properties). | |
```
Comment (michaelgriffiths-mclaren) [inline SignalR/2020.1/docs/ServiceConfig.md]: Extra `|` on the end here

File: SignalR/2020.1/docs/ServiceConfig.md [hunk 5]
```diff
@@ -0,0 +1,68 @@
...
+| TopicNames | An array of topic names to listen to for live data | `[ "Topic1", "Topic2" ]` |
+| Kafka | The Kafka broker configuration. [See below](#kafka-properties).
```
Comment (michaelgriffiths-mclaren) [inline SignalR/2020.1/docs/ServiceConfig.md]: Missing column end `|`

File: SignalR/2020.1/docs/ServiceConfig.md [hunk 6]
```diff
@@ -0,0 +1,68 @@
...
+| CorsOrigins         | List of addresses enabled for [cors]. Use "*" if you wish to allow all requests                |      ["*"]         | `["http://tap-api/explorer", "https://some-other-service"]` |
+
+<br>
+
+#### Kafka properties
+
+| Property  | Description                                                 | Default value | Example                                                                   |  
+|-|-|-|-|
+| DependencyService       |  The [dependency service] to publish dependencies to when replaying a session     |           |    `"http://dependency-service/api/dependencies/"`          |
+| BrokerList        | Address of the message broker cluster. Multiple can be specified using comma  |               |    `"127.0.0.1"`       |
```
Comment (michaelgriffiths-mclaren) [inline SignalR/2020.1/docs/ServiceConfig.md]: Does this require a port too?

File: SignalR/2020.1/docs/ServiceConfig.md [hunk 7]
```diff
@@ -0,0 +1,68 @@
...
+| ListenPort        | The port the server should listen to for requests made to `ListenAddress`.   |       `5000`        |   Any free port between 0 to 65535                                                    |  
+| CorsOrigins         | List of addresses enabled for [cors]. Use "*" if you wish to allow all requests                |      ["*"]         | `["http://tap-api/explorer", "https://some-other-service"]` |
+
+<br>
+
+#### Kafka properties
+
+| Property  | Description                                                 | Default value | Example                                                                   |  
+|-|-|-|-|
+| DependencyService       |  The [dependency service] to publish dependencies to when replaying a session     |           |    `"http://dependency-service/api/dependencies/"`          |
```
Comment (michaelgriffiths-mclaren) [inline SignalR/2020.1/docs/ServiceConfig.md]: Missing link with [dependency service] ?

File: SignalR/2020.1/docs/API.md (hunk 1 above)
Comment (PatrickMira) [reply SignalR/2020.1/docs/API.md]: Fixed thanks

File: SignalR/2020.1/docs/API.md (hunk 2 above)
Comment (PatrickMira) [reply SignalR/2020.1/docs/API.md]: Fixed

File: SignalR/2020.1/docs/ServiceConfig.md (hunk 4 above)
Comment (PatrickMira) [reply SignalR/2020.1/docs/ServiceConfig.md]: This is correct actually

File: SignalR/2020.1/docs/ServiceConfig.md (hunk 6 above)
Comment (PatrickMira) [reply SignalR/2020.1/docs/ServiceConfig.md]: Fixed

File: SignalR/2020.1/docs/ServiceConfig.md (hunk 7 above)
Comment (PatrickMira) [reply SignalR/2020.1/docs/ServiceConfig.md]: Fixed

File: SignalR/2020.1/docs/ServiceConfig.md (hunk 5 above)
Comment (PatrickMira) [reply SignalR/2020.1/docs/ServiceConfig.md]: Fixed

File: SignalR/2020.1/docs/API.md (hunk 3 above)
Comment (PatrickMira) [reply SignalR/2020.1/docs/API.md]: Fixed

**Comment by michaelgriffiths-mclaren** [comment]
As @peternagymclaren pointed out, this needs to be pointed at the 2020.1 branch instead

**Pull request merged**